Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages failed to build:</summary>
  <ul>
    <li>python311Packages.pyfunctional</li>
    <li>python311Packages.pyfunctional.dist</li>
    <li>python311Packages.vllm</li>
    <li>python311Packages.vllm.dist</li>
    <li>python312Packages.pyfunctional</li>
    <li>python312Packages.pyfunctional.dist</li>
    <li>python312Packages.vllm</li>
    <li>python312Packages.vllm.dist</li>
  </ul>
</details>
<details>
  <summary>16 packages built:</summary>
  <ul>
    <li>python311Packages.geoparquet</li>
    <li>python311Packages.geoparquet.dist</li>
    <li>python311Packages.outlines</li>
    <li>python311Packages.outlines.dist</li>
    <li>python311Packages.pyairports</li>
    <li>python311Packages.pyairports.dist</li>
    <li>python311Packages.python-ffmpeg</li>
    <li>python311Packages.python-ffmpeg.dist</li>
    <li>python312Packages.geoparquet</li>
    <li>python312Packages.geoparquet.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.pyairports</li>
    <li>python312Packages.pyairports.dist</li>
    <li>python312Packages.python-ffmpeg</li>
    <li>python312Packages.python-ffmpeg.dist</li>
  </ul>
</details>


```
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: pyfunctional
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<string>", line 1, in <lambda>
  File "/nix/store/pgb120fb7srbh418v4i2a70aq1w9dawd-python3-3.12.5/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'pyfunctional'
```

```
No ROCm runtime is found, using ROCM_HOME='/nix/store/1w7sl37p0xrf6i7m4jzwd6nyqhbg5a8l-clr-6.0.2'
<string>:65: RuntimeWarning: Failed to embed commit hash:
[Errno 2] No such file or directory: 'git'
Traceback (most recent call last):
  File "/nix/store/513rk7p8n17499jglnavq46pkz5k1ab3-python3.12-pyproject-hooks-1.0.0/lib/python3.12/site-packages/pyproject_hooks/_in_process/_in_>
    main()
  File "/nix/store/513rk7p8n17499jglnavq46pkz5k1ab3-python3.12-pyproject-hooks-1.0.0/lib/python3.12/site-packages/pyproject_hooks/_in_process/_in_>
    json_out['return_val'] = hook(**hook_input['kwargs'])
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/513rk7p8n17499jglnavq46pkz5k1ab3-python3.12-pyproject-hooks-1.0.0/lib/python3.12/site-packages/pyproject_hooks/_in_process/_in_>
    return hook(config_settings)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/llgbdr264r5ih0d19ii7k5c1ankxh9sv-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/build_meta.py", line 327,>
    return self._get_build_requires(config_settings, requirements=[])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/llgbdr264r5ih0d19ii7k5c1ankxh9sv-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/build_meta.py", line 297,>
    self.run_setup()
  File "/nix/store/llgbdr264r5ih0d19ii7k5c1ankxh9sv-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/build_meta.py", line 313,>
    exec(code, locals())
  File "<string>", line 472, in <module>
  File "<string>", line 393, in get_vllm_version
RuntimeError: Unknown runtime environment

ERROR Backend subprocess exited when trying to invoke get_requires_for_build_wheel
```
```

--- a/pkgs/development/python-modules/geoparquet/default.nix
+++ b/pkgs/development/python-modules/geoparquet/default.nix
@@ -34,7 +34,7 @@ buildPythonPackage {
   ];
 
   nativeCheckInputs = [ pytestCheckHook ];
-  pythonImportCheck = "geoparquet";
+  pythonImportsCheck = [ "geoparquet" ];
 
   doCheck = false; # no tests
 

--- a/pkgs/development/python-modules/pyfunctional/default.nix
+++ b/pkgs/development/python-modules/pyfunctional/default.nix
@@ -34,7 +34,7 @@ buildPythonPackage rec {
   ];
 
   nativeCheckInputs = [ pytestCheckHook ];
-  pythonImportCheck = "pyfunctional";
+  pythonImportsCheck = [ "functional" ];
 
   meta = {
     description = "Python library for creating data pipelines with chain functional programming";

--- a/pkgs/development/python-modules/python-ffmpeg/default.nix
+++ b/pkgs/development/python-modules/python-ffmpeg/default.nix
@@ -19,7 +19,7 @@ buildPythonPackage rec {
   propagatedBuildInputs = [ pyee ];
 
   nativeBuildInputs = [ setuptools-scm ];
-  pythonImportCheck = [ "ffmpeg" ];
+  pythonImportsCheck = [ "ffmpeg" ];
 
   meta = {
     homepage = "https://github.com/jonghwanhyeon/python-ffmpeg";

